Frontend/ui-driver/src/Components/NotificationDetailModel/View.purs

```diff
@@ -122,6 +122,13 @@ view push state =
                                           , height WRAP_CONTENT
                                           ]
                                       ]
+                                    , imageView
+                                      [ width MATCH_PARENT
+                                      , height $ V 150
+                                      , gravity CENTER
+                                      , imageUrl state.mediaUrl
+                                      , visibility if state.mediaType == Just ImageLink then VISIBLE else GONE
+                                      ]
```

Comment:
can we try to handle imageView height from native side using its height and width ratio from bitmap and then setting its height instead of hardcoding if possible

Reply:
@ashishsingh101  added the same !